addons/mail/static/tests/qunit_suite_tests/chatter_tests.js

```diff
@@ -262,6 +262,163 @@ QUnit.test('list activity exception widget with activity', async function (asser
     list.destroy();
 });
 
+QUnit.test('list activity widget: done the acitivty with "ENTER" keyboard shortcut', async function (assert) {
+    assert.expect(5);
+
+    const pyEnv = await startServer();
+    const [mailActivityTypeId1, mailActivityTypeId2] = pyEnv['mail.activity.type'].create([{}, {}]);
+    const [mailActivityId1, mailActivityId2] = pyEnv['mail.activity'].create([
+        {
+            display_name: "Call with Al",
+            date_deadline: moment().format("YYYY-MM-DD"), // now
+            can_write: true,
+            state: "today",
+            user_id: pyEnv.currentUserId,
+            create_uid: pyEnv.currentUserId,
+            activity_type_id: mailActivityTypeId1,
+        },
+        {
+            display_name: "Meet FP",
+            date_deadline: moment().add(1, 'day').format("YYYY-MM-DD"), // tomorrow
+            can_write: true,
+            state: "planned",
+            user_id: pyEnv.currentUserId,
+            create_uid: pyEnv.currentUserId,
+            activity_type_id: mailActivityTypeId2,
+        }
+    ]);
+    pyEnv['res.users'].write([pyEnv.currentUserId], {
+        activity_ids: [mailActivityId1, mailActivityId2],
+        activity_state: 'today',
+        activity_summary: 'Call with Al',
+        activity_type_id: mailActivityTypeId2,
+    });
+
+    const { widget: list } = await start({
+        hasView: true,
+        View: ListView,
+        model: 'res.users',
+        arch: `
+            <list>
+                <field name="activity_ids" widget="list_activity"/>
+            </list>`,
+        mockRPC: function (route, args) {
+            if (!['/mail/init_messaging', '/mail/load_message_failures'].includes(route)) {
+                assert.step(args.method || route);
+            }
+            if (args.method === 'action_feedback') {
+                pyEnv['res.users'].write([pyEnv.currentUserId], {
+                    activity_ids: [mailActivityId2],
+                    activity_state: 'planned',
+                    activity_summary: 'Meet FP',
+                    activity_type_id: mailActivityTypeId1,
+                });
+                return Promise.resolve();
+            }
+            return this._super(route, args);
+        },
+        intercepts: {
+            switch_view: () => assert.step('switch_view'),
+        },
+    });
+
+    await testUtils.dom.click(list.$('.o_activity_btn span'));
+    await testUtils.dom.click(list.$('.o_mark_as_done:first'));
+    await testUtils.dom.triggerEvent(
+        document.querySelector('.o_activity_popover_done'),
+        'keydown',
+        { key: 'Enter' },
+    );
+    assert.verifySteps([
+        '/web/dataset/search_read',
+        'activity_format',
+        'action_feedback',
+        'read',
+    ]);
+
+    list.destroy();
+});
+
+QUnit.test('list activity widget: done and schedule the next acitivty with "ENTER" keyboard shortcut', async function (assert) {
```

Comment:
Exact same remarks in this test than in the previous test:
- typo in name of field
- too many activity data: keep a single activity (type)
- replace `assert.step()`/`assert.verifySteps` by actual assertion on visual Ui that activity is done and schedule next
- implement `action_feedback_schedule_next` in MockServer so that you don't have to cheat in the test.